apps/src/lib/node/ledger/events.rs

```diff
@@ -102,9 +102,16 @@ impl Event {
         event
     }
 
+    /// Check if the events keys contains a given string
     pub fn contains_key(&self, key: &str) -> bool {
         self.attributes.contains_key(key)
     }
+
+    /// Get the value corresponding to a given key, if it exists.
+    /// Else return None.
+    pub fn get(&self, key: &str) -> Option<&String> {
```

Comment:
should we return `Option<&str>` here instead somehow?

Reply:
We could, but it doesn't make a real difference.